paper-input-container.html

```diff
@@ -594,12 +610,21 @@
           if (label) {
             this.$.labelAndInputContainer.style.position = 'relative';
           }
+          if (invalid) {
+            cls += ' is-invalid';
+          }
         }
       } else {
         if (_inputHasContent) {
           cls += ' label-is-hidden';
         }
+        if (invalid) {
```

Comment:
I put this here to cover the case that the input is invalid because it's empty. When I required field was empty the `_inputHasContent` check was preventing the error state.